cafebabel/static/css/styles.css

```diff
@@ -1554,6 +1554,7 @@ select#author {
 
 .article-page article > img {
   margin: 0 auto;
+  width: 100%; /* force archive images (1180px wide) to be 1200px */
```

Comment:
Isn't it more pertinent to keep 1180px to avoid blurriness?

Reply:
First I hate this random 1180 number (not a round number: quite fussy I know), and second I set the content max-width to 1200px everywhere cause it's very common and it makes more sense to me as well. What do you think? I know it's very bad to stretch images, but for 20px, I don't care. Really.